qemu/tests/qemu_disk_img_info.py

```diff
@@ -50,9 +49,38 @@ def run(test, params, env):
     :param env: Dictionary with test environment.
     """
     base_image = params.get("images", "image1").split()[0]
-    params.update(
-        {"image_name_%s" % base_image: params["image_name"],
-         "image_format_%s" % base_image: params["image_format"]})
+
+    update_params = {
+        "image_name_%s" % base_image: params["image_name"],
+        "image_format_%s" % base_image: params["image_format"]
+    }
+
+    optval = (lambda opt, img, p, default: p.get('%s_%s' % (opt, img),
```

Comment:
Please define a function instead of assigning `lambda` function to a variable.

Reply:
Confirmed with lolyu, for the pep8 checker ignored the warning when use (), leave code unchanged.